src/core/scene/a-scene.js

```diff
@@ -123,37 +143,64 @@ module.exports = registerElement('a-scene', {
       }
     },
 
-    /**
-     * Generally must be triggered on user action for requesting fullscreen.
-     */
     enterVR: {
       value: function (event) {
         var self = this;
-        return this.effect.requestPresent().then(enterVRSuccess, enterVRFailure);
+        if (this.is('vr-mode')) { return; }
+        if (checkHeadsetConnected() || this.isMobile) {
+          // We send the canvas fullscreen so mirroring overlays
+          // any other elements on the page. Important with
+          // embedded scenes.
+          if (!this.isMobile) { this.requestFullscreen(); }
```

Comment:
This might need to be in a separate `if` clause since it's weird to have `if (connected || mobile) { if (not mobile) ...`